libr/bin/format/elf/elf.c

```diff
@@ -1014,6 +1014,11 @@ ut64 Elf_(r_bin_elf_get_baddr)(struct Elf_(r_bin_elf_obj_t) *bin) {
 			}
 		}
 	}
+	if (base == UT64_MAX  && bin->ehdr.e_type == ET_REL) {
+		//we return our own base address for ET_REL type
+		//we act as a loader for ELF
+		return 0x08000000;
```

Comment:
That's pretty nifty, and works well on the couple of examples I ran this on ;)

As for sharing information - I'm more concerned about gnu tools than IDA/Hopper, but that's just me ;)
Since we can still retrieve the physical addresses of virtual ones, I guess we can have a common reference, so that's fine ;)